aws/resource_aws_transfer_server.go

```diff
@@ -150,6 +172,14 @@ func resourceAwsTransferServerCreate(d *schema.ResourceData, meta interface{}) e
 		createOpts.LoggingRole = aws.String(attr.(string))
 	}
 
+	if attr, ok := d.GetOk("certificate"); ok {
+		createOpts.Certificate = aws.String(attr.(string))
+	}
+
+	if v, ok := d.GetOk("protocols"); ok {
+		createOpts.Protocols = expandStringSet(v.(*schema.Set))
+	}
+
```

Comment:
Need to check for update of `certificate` and `protocols` in `resourceAwsTransferServerUpdate()` also.

Reply:
indeed, i've compiled this PR locally to use it, and i noticed it didn't update the certificate after I changed it once.